packages/lambda-tiler/src/index.ts

```diff
@@ -19,6 +20,10 @@ export const handler = lf.http(LogConfig.get());
 handler.router.hook('request', (req) => {
   req.set('name', 'LambdaTiler');
 
+  // Set the asset location for asset provider
+  const assetLocation = Env.get(Env.AssetLocation);
+  assetProvider.set(assetLocation);
```

Comment:
you made this async, it doesnt need to be, but if its async you need to `await`

Reply:
Good pick.